### ISSUE TITLE
Ionic2 <ion-select> onclick not working.

### ISSUE THREAD

**Issue by AjkP** [driftyco/ionic#6615]
I tried many things and at the end I even copied the template from the preview app.
https://github.com/driftyco/ionic-preview-app/blob/master/app/pages/selects/basic/pages.ts

Below the code:
HTML:
```
<ion-content>
  <ion-list>
    <ion-item>
      <ion-label>Gender</ion-label>
      <ion-select [(ngModel)]="gender">
        <ion-option value="f">Female</ion-option>
        <ion-option value="m">Male</ion-option>
      </ion-select>
    </ion-item>
  </ion-list>
</ion-content>
```

TS:
```
// Page template
@Page({
    templateUrl: 'build/pages/settings/settings.html'
})

export class SettingsPage {
    gender: string = "f";
}
```

The model preset selection (Female) appears, but the selection is not clickable.

When I click on the select widget I do not get the alert modal. I tried it on Chrome, Chrome with Ripple and iPhone 6s plus simulator.

Attached a screenshot.
<img width="206" alt="screen shot 2016-05-22 at 5 50 58 pm" src="https://cloud.githubusercontent.com/assets/6978905/15457814/d6a9477c-2045-11e6-9a08-8bf7b405485e.png">

```
➜  ~  ionic -v
2.0.0-beta.23
```

**Comment by brandyscarney** [comment]
Do you have any errors in the Chrome developer tools console? Also, does your app have an `ion-nav` element in the main app template?

Can you run `ionic info` from within the project and paste the output, too. Thanks!

**Comment by AjkP** [comment]
>Do you have any errors in the Chrome developer tools console?
No errors in the console. There is a few warning/infos. Posting below:
```
DEPRECATION WARNING: 'enqueueTask' is no longer supported and will be removed in next major release. Use addTask/addRepeatingTask/addMicroTask
lang.ts:389 Angular 2 is running in the development mode. Call enableProdMode() to enable the production mode.
angular2-polyfills.js:1152 DEPRECATION WARNING: 'dequeueTask' is no longer supported and will be removed in next major release. Use removeTask/removeRepeatingTask/removeMicroTask
plugin.js:42 Native: tried calling StatusBar.styleDefault, but Cordova is not available. Make sure to include cordova.js or run in a device/simulator
VM536:1 Deferred long-running timer task(s) to improve scrolling smoothness. See crbug.com/574343.
```

> Also, does your app have an ion-nav element in the main app template?
It does although I have tried to remove it with no change in behavior. Posting ion-nav below.
```
<ion-navbar *navbar>
  <ion-title>About</ion-title>
</ion-navbar>
```

```
WARN: ionic.config.js has been deprecated, you can remove it.

Your system information:

Cordova CLI: 6.0.0
Ionic Version: 2.0.0-beta.3
Ionic CLI Version: 2.0.0-beta.23
Ionic App Lib Version: 2.0.0-beta.13
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Mac OS X El Capitan
Node Version: v5.9.1
Xcode version: Xcode 7.2 Build version 7C68

******************************************************
 Dependency warning - for the CLI to run correctly,
 it is highly recommended to install/upgrade the following:

 Install ios-sim to deploy iOS applications. `npm install -g ios-sim` (may require sudo)
 Install ios-deploy to deploy iOS applications to devices. `npm install -g ios-deploy` (may require sudo)

******************************************************
```

**Comment by jgw96** [comment]
Hello @AjkP! Hmm it seems that your on an older beta of Ionic 2 and an old version of the Ionic@beta cli. Would you mind updating both your Ionic CLI (using npm install -g ionic@beta) and then upgrade your project to the latest version of Ionic 2 using these instructions here: https://github.com/driftyco/ionic/blob/2.0/CHANGELOG.md#angular-update-to-200-rc1 and seeing if you still have the issue? Thanks for using Ionic!

**Comment by brandyscarney** [comment]
@AjkP Sorry I should have been more specific, I was responding from my phone. 😛  There should be an `<ion-nav>` element somewhere in your main template, it is required for the overlays to work. Something like this:

```
<ion-nav [root]="root" #content swipe-back-enabled="false"></ion-nav>
```

However, like Justin said, it would be good to update your CLI and framework versions just to make sure everything is up to date. It may be easier to create a new project though and copy over your app's code since there were so many changes from `beta.3` to `beta.7`. 

The latest version of the CLI is `beta.25` and the latest version of the framework is `beta.7` so make sure to update the CLI before starting a new project so you get a `beta.7` project. :smile:

**Comment by AjkP** [comment]
@brandyscarney - My app does not have a `ion-nav`, instead it uses tabs as below:
```
<ion-tabs>
    <ion-tab tabIcon="school" tabTitle="Practice" [root]="tabPractice" [rootParams]="tabPracticeParams"></ion-tab>
    <ion-tab tabIcon="paper" tabTitle="Test" [root]="tabRealTest" [rootParams]="tabRealTestParams"></ion-tab>
    <ion-tab tabIcon="contact" tabTitle="Support" [root]="tabContactUs" ></ion-tab>
    <ion-tab tabIcon="settings" tabTitle="Settings" [root]="tabSettings" ></ion-tab>
</ion-tabs>
```

Using a simple ion-nav seemed to get the widget to work 😮 
```
<ion-nav id="nav" [root]="tabSettings" swipe-back-enabled="false"></ion-nav>
```

But it removed the tabs. I ll try to encapsulate the tabs in a different page. I guess I don't quite understand the importance of the `ion-nav` element.

@jgw96 I'll make sure I update.

**Comment by brandyscarney** [comment]
Yeah, currently the `<ion-nav>` is required for all elements that use overlays (alert, action-sheet, modal, select, etc.). 

The conference app has an example of using `<ion-nav>` with tabs. Instead of setting the root page to `TutorialPage`, you would set it to the `TabsPage`:

https://github.com/driftyco/ionic-conference-app/blob/master/app/app.html#L69
https://github.com/driftyco/ionic-conference-app/blob/master/app/app.js#L50

**Comment by AjkP** [comment]
Thanks @brandyscarney . That app is super cool! Maybe it could be referenced from the tutorial / documentation somewhere?

Closing issue.
